### ISSUE TITLE
Photo capture fail in radu mode in ESP32-CAM-Video-Telegram_9.7 

### ISSUE THREAD

**Issue by hakam007** [jameszah/ESP32-CAM-Video-Telegram#51]
Sir, I am facing this issue sometimes( not all the time) during Radu mode enabled when I send /photo or /fphoto command ,but during pir event in Radu mode no pic issue ....thanks

**Comment by jameszah** [comment]
Yes, radu mode is very poor ... I did a another version somewhat better ... just going to get started on it again.

**Comment by hakam007** [comment]
Thankyou sir ,really great work done by you,  esp cam will be highly useful for night .

**Comment by FBMinis** [comment]
> Yes, radu mode is very poor ... I did a another version somewhat better ... just going to get started on it again.

Just read your reply. Are you working on a new version?
